src/arch/xtensa/include/arch/drivers/interrupt.h

```diff
@@ -14,7 +14,7 @@
 #include <xtensa/xtruntime.h>
 #include <stddef.h>
 #include <stdint.h>
-#include <config.h>
+
 
```

Comment:
It looks like you are removing the include but adding a newline. Can you just remove the include without adding the new line?

Reply:
I can but this would break the main test, see commit message.